soccer/Robot.hpp

```diff
@@ -530,6 +532,9 @@ class OurRobot : public Robot {
     friend class MotionControl;
 
 private:
+    mutable RJ::Time _lastBallSense;
```

Comment:
I decided on mutable instead of removing const because hasBall intuitively seems like a const function. But yeah I'm unsure of what's better coding style